modules/store/testing/spec/mock_store.spec.ts

```diff
@@ -281,6 +291,101 @@ describe('Mock Store', () => {
   });
 });
 
+describe('Mock Store with Injector', () => {
+  let injector: Injector;
+  const initialState = { counter: 0 };
+  const mockSelector: MockSelector = { selector: 'counter', value: 10 };
```

Comment:
You'll need to change to this one, if you'll be using `mockStore = getMockStore({ initialState, selectors: [mockSelector] });` instead.

```suggestion
   const mockSelector = { selector: 'counter', value: 10 } as const;
```

Reply:
done